python_modules/dagster/dagster/_core/execution/context/compute.py

```diff
@@ -1300,15 +1305,24 @@ def typed_event_stream_error_message(self) -> Optional[str]:
     def set_requires_typed_event_stream(self, *, error_message: Optional[str] = None) -> None:
         self._step_execution_context.set_requires_typed_event_stream(error_message=error_message)
 
+    @staticmethod
+    def get_current() -> Optional["OpExecutionContext"]:
```

Comment:
should these raise or return None when there is no current context?

Reply:
IMO we should raise